video/src/backend.rs

```diff
@@ -0,0 +1,68 @@
+use crate::error::Error;
+use crate::frame::{EncodedFrame, FrameDependency};
+use crate::VideoStreamHandle;
+use ruffle_render::backend::RenderBackend;
+use ruffle_render::bitmap::BitmapInfo;
+use swf::{VideoCodec, VideoDeblocking};
+
+/// A backend that provides access to some number of video decoders.
+///
+/// Implementations of `VideoBackend` are not required to actually support
+/// decoding any video formats. However, they must interoperate with at least
+/// one `RenderBackend` so that renderable video frames may be passed from the
+/// decoder to the renderer.
+pub trait VideoBackend {
```

Comment:
Since the backends are the main type of the backend crates, I wonder if this should be re-exported at the top-level, so that it's `use ruffle_video::VideoBackend;` instead of `use ruffle_video::backend::VideoBackend`? Possibly for a handful of other types too.

Maybe for a future PR since the render backends are set up with `backend::RenderBackend` as well.

Reply:
Yeah that makes sense!